bsp/esp32_p4_function_ev_board/include/bsp/esp-bsp.h

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+#include "bsp/esp32_p4.h"
```

Comment:
Usually, the main header name is the same as BSP name. Would it be better to name it `bsp/esp32_p4_function_ev_board.h` here?